app/src/main/res/values/strings.xml

```diff
@@ -34,6 +34,21 @@
     <string name="zoom_in">Zoom in</string>
     <string name="zoom_out">Zoom out</string>
 
+    <!-- Hike Details Screen -->
+    <string name="hike_detail_screen_bookmark_hint_on_isSaved_true">Unsave hike</string>
+    <string name="hike_detail_screen_bookmark_hint_on_isSaved_false">Save hike</string>
+    <string name="hike_detail_screen_label_distance">Distance</string>
+    <string name="hike_detail_screen_label_elevation_gain">Elevation Gain</string>
+    <string name="hike_detail_screen_label_estimated_time">Estimated time</string>
+    <string name="hike_detail_screen_label_difficulty">Difficulty</string>
+    <string name="hike_detail_screen_label_status">Status</string>
+    <string name="hike_detail_screen_label_planned_for">Planned for</string>
+    <string name="hike_detail_screen_value_saved">Saved</string>
+    <string name="hike_detail_screen_value_not_saved">Not saved</string>
+    <string name="hike_detail_screen_value_planned">Planned</string>
+    <string name="hike_detail_screen_add_a_date_button_text">Add a date</string>
+    <string name="hike_detail_screen_value_default_date">dd/mm/yyy</string>
```

Comment:
Yes, that was a placeholder since the vM can not get the date just yet. I was just copy pasting all the things into the string resource file without thinking. I've replaced it to be a string again and put a comment in the code, as soon as the vM is updated we can display the real date.